packages/clay-card/src/types.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+export type CardDescriptionDisplayType = 'text' | 'title' | 'subtitle';
+
+export type CardDisplayType =
+	| 'directory'
```

Comment:
This might have to be `horizontal` to maintain consistency with `horizontal interactive`.

Reply:
Makes sense, changing.